studio/src/editors/merch-card-editor.js

```diff
@@ -7,7 +7,7 @@ import '../aem/aem-tag-picker-field.js';
 import './variant-picker.js';
 import { SPECTRUM_COLORS } from '../utils/spectrum-colors.js';
 import '../rte/osi-field.js';
-import { CARD_MODEL_PATH } from '../constants.js';
+import { CARD_MODEL_PATH } from '../constants.js';
```

Comment:
I don't understand this change

Reply:
Me neither 😅 I think it was generated by prettier automatically when I saved the file.

Reply:
It's gone after merging with main